tags.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Tags
+hide: true
+---
+
+{% capture site_tags %}{% for tag in site.tags %}{{ tag | first }}{% unless forloop.last %},{% endunless %}{% endfor %}{% endcapture %}
+{% assign tag_words = site_tags | split:',' | sort %}
+<div class="home">
+  <!--  cycles through tag list and creates header row of all tags used in site with accompanying per-tag counts -->
+  {% for item in (0..site.tags.size) %}
+    {% unless forloop.last %}
+      {% capture this_word %}{{ tag_words[item] | strip_newlines }}{% endcapture %}
+      <a class="button" href="#{{ this_word | cgi_escape }}">
```

Comment:
Button margin isn't consistent with that on the post pages. Please make this `tags.html` page have the same `margin-right` on the buttons as the post pages.